Blizzard.IO.RabbitMQ/NetqRabbitConfigurator.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using EasyNetQ.Topology;
+using Microsoft.Extensions.Logging;
+
+namespace Blizzard.IO.RabbitMQ
+{
+    public class NetqRabbitConfigurator : IRabbitConfigurator
+    {
+        private readonly IBus _netqBus;
+        private readonly ILogger<NetqRabbitConfigurator> _logger;
+
+        public NetqRabbitConfigurator(IBus netqBus, ILogger<NetqRabbitConfigurator> logger)
```

Comment:
Pass a logger factory like you did in the consumer

Reply:
f